stable/pgadmin/templates/deployment.yaml

```diff
@@ -54,6 +54,8 @@ spec:
             {{- .Values.readinessProbe | toYaml | nindent 12 }}
         {{- end }}
           env:
+            - name: PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION
+              value: !!string {{ .Values.env.enhanced_cookie_protection }}
```

Comment:
I just checked here: https://www.pgadmin.org/docs/pgadmin4/latest/container_deployment.html

and it looks like this doesn't have to be a string at all, e.g.:
```    -e 'PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION=True' \```

Reply:
helm install .                                                                                                                                                                                                                                                     ─╯
Error: release tinseled-deer failed: Deployment in version "v1" cannot be handled as a Deployment: v1.Deployment.Spec: v1.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Env: []v1.EnvVar: v1.EnvVar.Value: ReadString: expects " or n, but found f, error found in #10 byte of ...|,"value":false},{"na|..., bigger context ...|ADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION","value":false},{"name":"PGADMIN_DEFAULT_EMAIL","value":"cha|...

if i dont force to string it does not work 

another solution could be 

              value: \"{{ .Values.env.enhanced_cookie_protection }}\"

Reply:
- but the pod will not start because  it will be transformed to 
    ENHANCED_COOKIE_PROTECTION = \"false\"

Reply:
the problem come from the Deployment yaml definition 
if we dont forced to String we will got this:

````
Error: release pgadmin failed: Deployment in version "v1" cannot be handled as a Deployment: v1.Deployment.Spec: v1.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Env: []v1.EnvVar: v1.EnvVar.Value: ReadString: expects " or n, but found f, error found in #10 byte of ...|,"value":false},{"na|..., bigger context ...|ADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION","value":false},{"name":"PGADMIN_DEFAULT_EMAIL","value":"cha
````

the other way to transforme it will be:
````
 env:
            - name: PGADMIN_CONFIG_ENHANCED_COOKIE_PROTECTION
              value: \"{{ .Values.env.enhanced_cookie_protection }}\"
`````
the issue in the log that you showed come from the transformation of False to false
because I put:

`````
  enhanced_cookie_protection: False

`````
it should be :

`````
  enhanced_cookie_protection: "False"

`````